packages/utils/src/isAscii.ts

```diff
@@ -0,0 +1,4 @@
+export function isAscii(value?: string): boolean {
+    if (!value) return true;
+    return /^[\x00-\x7F]*$/.test(value); // eslint-disable-line
```

Comment:
nit: I would prefer disable only that one rule that is throwing error

Reply:
Nice detail! Fixed! :+1: